Library/Homebrew/test/Rakefile

```diff
@@ -1,8 +1,14 @@
 require "rake"
 require "rake/testtask"
 
+def mac?
+  /darwin/i === RUBY_PLATFORM && !ENV["HOMEBREW_TEST_GENERIC_OS"]
```

Comment:
Yes, the `include?` check is even better. I've never seen a different capitalization of `darwin` and the case-insensitive match wasn't actually necessary, though I'm not 100% sure (maybe back in the old days, way before my time …).